scripts/variantstore/wdl/GvsCreateAltAllele.wdl

```diff
@@ -27,8 +29,9 @@ workflow GvsCreateAltAllele {
 
   call Utils.GetBQTableLastModifiedDatetime {
     input:
+      go = CreateAltAlleleTable.done,
```

Comment:
bug 2: getting the last modified datetime of the alt allele table should run only after the alt allele table has been created